build/media-suite_compile.sh

```diff
@@ -1750,14 +1750,34 @@ if [[ $ffmpeg != "no" ]]; then
                 do_removeOption --enable-libsvthevc
         fi
 
+        # -- SVT-AV1 patches for ffmpeg --
+
+        # See issue https://github.com/OpenVisualCloud/SVT-AV1/issues/567 for the reasons behind the changes below.
+        # This is broken in the meantime. Either libsvthevc or libsvtav1 can be enabled, but not both at the same time.
+        #remove the do_removeOption line and uncomment the patch once the issue have been fixed.
         if enabled libsvthevc && enabled libsvtav1; then
```

Comment:
Can you move this check to before applying svt-hevc's patch?

Reply:
done.